files/es/web/javascript/reference/global_objects/symbol/hasinstance/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Symbol.hasInstance
+slug: Web/JavaScript/Reference/Global_Objects/Symbol/hasInstance
+browser-compat: javascript.builtins.Symbol.hasInstance
+---
+{{JSRef}}
+
+El símbolo conocido **`Symbol.hasInstance`** se utiliza para determinar si un objeto constructor reconoce un objeto como su instancia. El comportamiento del operador {{jsxref("Operators/instanceof", "instanceof")}} puede personalizarse mediante este símbolo.
```

Comment:
```suggestion
El símbolo conocido como **`Symbol.hasInstance`** se utiliza para determinar si un objeto constructor reconoce un objeto como su instancia. El comportamiento del operador {{jsxref("Operators/instanceof", "instanceof")}} puede personalizarse mediante este símbolo.
```

Reply:
Este en específico lo noté, pero seguí leyendo en busca de otros problemas, arrelé los otros y olvidé ese, gracias por la corrección.